readthedocs/projects/views/public.py

```diff
@@ -47,11 +47,8 @@ def get_queryset(self):
         queryset = Project.objects.public(self.request.user)
         queryset = queryset.exclude(users__profile__banned=True)
 
-        if self.kwargs.get('tag'):
-            self.tag = get_object_or_404(Tag, slug=self.kwargs.get('tag'))
-            queryset = queryset.filter(tags__slug__in=[self.tag.slug])
-        else:
-            self.tag = None
+        self.tag = get_object_or_404(Tag, slug=self.kwargs.get('tag'))
+        queryset = queryset.filter(tags__slug__in=[self.tag.slug])
 
         if self.kwargs.get('username'):
```

Comment:
It doesn't seem like this is used anymore. As a separate change in a separate PR, this could probably be removed.

Reply:
#6250